sql/gis/rtree_support.h

```diff
@@ -91,6 +91,21 @@ bool mbr_contain_cmp(const dd::Spatial_reference_system *srs, rtr_mbr_t *a,
 bool mbr_equal_cmp(const dd::Spatial_reference_system *srs, rtr_mbr_t *a,
                    rtr_mbr_t *b);
 
+/// Checks if two MBRs are equal more precisely
+///
+/// For both MBRs, the coordinates of the MBR's minimum corners must be smaller
```

Comment:
Perhaps it is worth to add asserts enforcing this similarly to those we have in mbr_equal_cmp() ?